Publish release notes for soft-released Godot 4.1 RC 2 (#661)
godotengine/godot-website: collections/_article/release-candidate-godot-4-1-rc-2.md

---
title: "Release candidate: Godot 4.1 RC 2"
excerpt: "A few more regressions were fixed, and we should now be ready for the Godot 4.1 release. Help us confirm that by testing RC 2!"
categories: ["pre-release"]
author: Rémi Verschelde
image: /storage/blog/covers/release-candidate-godot-4-1-rc-2.webp
image_caption_title: "Skies above the Great War"
image_caption_description: "A game by Puntigames"
date: 2023-06-30 09:00:00
---

Godot 4.1 is shaping up to be ready [right on schedule](/article/release-management-4-1). Over the course of the beta testing phase, and with our first [Release Candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate) a few days ago, we identified and fixed a number of regressions, i.e. recently introduced bugs that would worsen the experience compared to Godot 4.0. Now we're fairly confident that we've handled most of those, and the remaining issues we're tracking will likely be deferred to a fix in future 4.1.x maintenance releases.

> That is to say, this RC 2 should be the one – if no major regression is identified in coming days, we'll go ahead and release 4.1 stable, and reopen the development cycle for 4.2, which is scheduled for a release in November 2023. Please keep reporting issues that you encounter with the RC, even if you don't think they're critical – while triaging them, we might prioritize them for early on in the 4.2 development cycle, and backporting the fixes to 4.1.x.

Reviewer: ```suggestion
That is to say, this RC 2 should be the one – if no major regression is identified in coming days, we'll go ahead and release 4.1 stable, and reopen the development cycle for 4.2, which is scheduled for a release in November 2023. Please keep reporting issues that you encounter with the RC, even if you don't think they're critical – while triaging them, we might prioritize them for early on in the 4.2 development cycle, and backport the fixes to 4.1.x.
```

I think? :)

Author: Yes, I should proofread myself :D

---
title: "Release candidate: Godot 4.1 RC 2"
excerpt: "A few more regressions were fixed, and we should now be ready for the Godot 4.1 release. Help us confirm that by testing RC 2!"
categories: ["pre-release"]
author: Rémi Verschelde
image: /storage/blog/covers/release-candidate-godot-4-1-rc-2.webp
image_caption_title: "Skies above the Great War"
image_caption_description: "A game by Puntigames"
date: 2023-06-30 09:00:00
---

> Godot 4.1 is shaping up to be ready [right on schedule](/article/release-management-4-1). Over the course of the beta testing phase, and with our first [Release Candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate) a few days ago, we identified and fixed a number of regressions, i.e. recently introduced bugs that would worse the experience compared to Godot 4.0. Now we're fairly confident that we've handle most of those, and the remaining issues we're tracking will likely be deferred to a fix in future 4.1.x maintenance releases.

Reviewer: ```suggestion
Godot 4.1 is shaping up to be ready [right on schedule](/article/release-management-4-1). Over the course of the beta testing phase, and with our first [Release Candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate) a few days ago, we identified and fixed a number of regressions, i.e. recently introduced bugs that would worsen the experience compared to Godot 4.0. Now we're fairly confident that we've handled most of those, and the remaining issues we're tracking will likely be deferred to a fix in future 4.1.x maintenance releases.
```

> That is to say, this RC 2 should be the one – if no major regression is identified in coming days, we'll go ahead and release 4.1 stable, and reopen the development cycle for 4.2, which is scheduled for a release in November 2023. Please keep reporting issues that you encounter with the RC, even if you don't think they're critical – we might triage not fix them for the imminent 4.1 release, but they may be prioritized to fix early on in the 4.2 development cycle, and backported to 4.1.x.

Reviewer: ```suggestion
That is to say, this RC 2 should be the one – if no major regression is identified in coming days, we'll go ahead and release 4.1 stable, and reopen the development cycle for 4.2, which is scheduled for a release in November 2023. Please keep reporting issues that you encounter with the RC, even if you don't think they're critical – we might decide not to fix them for the imminent 4.1 release, but they can be prioritized to be fixed early on in the 4.2 development cycle, and backported to 4.1.x.
```

This doesn't read right, but you can go with something else rather my suggestion.

## Downloads

The downloads for this dev snapshot can be found directly on our repository:

* [Standard build](https://downloads.tuxfamily.org/godotengine/4.1/rc2/) (GDScript, GDExtension).
* [.NET 6 build](https://downloads.tuxfamily.org/godotengine/4.1/rc2/mono) (C#, GDScript, GDExtension).
  - Requires [.NET SDK 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) or [7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) installed in a standard location.

## Known issues

With every release we accept that there are going to be various issues, which have already been reported but haven't been fixed yet. See the GitHub issue tracker for a list of [known bugs in the 4.1 milestone](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aopen+milestone%3A4.1+label%3Abug+).

Some notable issues are still tracked for a potential fix before 4.1-stable:
- The processing order for internal and normal node processing was mistakenly changed, causing issues such as [GH-77548](https://github.com/godotengine/godot/issues/77548). You can help us test [GH-78745](https://github.com/godotengine/godot/pull/78745) to validate that it solves the issue without further regression.

> - The editor output dock doesn't update its scrollbar properly in some cases ([GH-78434](https://github.com/godotengine/godot/issues/78434)). A potential solution is identified but we're not sure yet that it's safe to include in 4.1 at this late stage given the risk of regression.

Reviewer: Both have been merged in RC2 🙃
